src/main/java/uk/gov/hmcts/cmc/claimstore/services/FormaliseResponseAcceptanceService.java

```diff
@@ -80,14 +99,20 @@ private void createEventForReferToJudge(Claim claim, String authorisation) {
         Response response = claim.getResponse()
             .orElseThrow(() -> new IllegalArgumentException(MISSING_RESPONSE));
         CaseEvent caseEvent;
+        Claim updatedClaim = claim;
         if (isCompanyOrOrganisation(response.getDefendant())) {
             eventProducer.createRejectOrganisationPaymentPlanEvent(claim);
+            User user = userService.authenticateAnonymousCaseWorker();
+            PDF document = claimantResponseReceiptService.createPdf(claim,
```

Comment:
can use the same private method **uploadInterlocutoryJudgmentDocumentToDocumentStore**